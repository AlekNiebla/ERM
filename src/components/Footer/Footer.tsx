import * as React from "react";

const Footer = () => <span>Footer</span>;

export default Footer;
