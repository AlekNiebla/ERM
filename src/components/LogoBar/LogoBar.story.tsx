import * as React from "react";
import { storiesOf } from "@storybook/react";
import LogoBar from "./LogoBar";
import Button from "Components/Button/Button";

storiesOf("Components / LogoBar", module)
  .add("default", () => <LogoBar />)
  .add("with additional elements", () => (
    <LogoBar>
      <Button label="Hello" />
    </LogoBar>
  ));
