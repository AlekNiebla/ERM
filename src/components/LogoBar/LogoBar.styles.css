.main {
  display: flex;
  align-items: center;
}

.logo {
  composes: Typo-Big from global;
  font-weight: bold;
  font-family: sans-serif;
  color: var(--color-negative);
}
