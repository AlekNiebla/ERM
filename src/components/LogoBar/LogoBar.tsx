import * as React from "react";
// css
import * as styles from "./LogoBar.styles.css";
// interface
interface ILogoBar {
  button?: React.ReactNode;
}

const LogoBar = (props: ILogoBar) => (
  <div className={styles.main}>
    <span className={styles.logo}>netflixroulette</span>
    {props.button && <div>{props.button}</div>}
  </div>
);

export default LogoBar;
