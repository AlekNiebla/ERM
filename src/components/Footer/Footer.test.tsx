import * as React from "react";
import { shallow } from "enzyme";
// components
import Footer from "./Footer";
import LogoBar from "./../LogoBar/LogoBar";

describe("Footer component...", () => {
  it("...renders without errors", () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper.find(Footer)).toBeTruthy();
    expect(wrapper.find(LogoBar)).toBeTruthy();
    // snapshot
    expect(wrapper).toMatchSnapshot();
  });
});
