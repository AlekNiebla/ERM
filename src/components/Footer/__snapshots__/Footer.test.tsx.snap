// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component... ...renders without errors 1`] = `ShallowWrapper {}`;
